daemon/codec.c

```diff
@@ -2485,7 +2488,11 @@ void codec_output_rtp(struct media_packet *mp, struct codec_scheduler *csch,
 
 	ts_diff_us = p->ttq_entry.when - rtpe_now;
 
-	csch->output_skew = csch->output_skew * 15 / 16 + ts_diff_us / 16;
+	if (csch->is_media_playback) {
+		ilogs(transcoding, LOG_DEBUG, "Skipping clock skew calculation during media playback");
```

Comment:
This would be printed for every single packet if debugging is enabled. Either add the `LOG_FLAG_LIMIT` modifier or just remove it (maybe just log it once when the flag is set)

Reply:
good shout, was handy while i was testing but not needed now. have added a line when the flag is set instead